README.md

```diff
@@ -32,6 +28,21 @@ const account = new Account(Tozny, 'http://platform.local.tozny.com:8000') // wh
 
 The `Account` instance provides methods for creating a account client in various ways. The client object returned from each is identical in use and operation, but the method matches a context specific means of creating the client. See the [`Account` class definition](src/account.js) for the full list of available methods and return values.
 
+### Migration to Typescript
+
+Originally, this project used vanilla ES6 javascript that is natively available in Node 6+. Beginning in July 2021, we started a process of migrating to typescript for the added type safety and error checking. Transition to to typescript is a work in progress. All new code added to this library should be written in typescript. Existing modules are converted as needed.
+
+Source code in `src/` is compiled from ts/js to ES6 js in `dist/`.
```

Comment:
Suggest a note here about why we check in the intermediate ES6 code to source control